samples/host_verify/host.c

```diff
@@ -110,12 +115,23 @@ oe_result_t verify_report(
                 &endorsement_file_size);
         }
 
-        result = oe_verify_remote_report(
+        oe_verifier_initialize();
+        result = oe_verify_evidence(
+            &_uuid_legacy_report_remote,
             report_data,
             report_file_size,
             endorsement_data,
             endorsement_file_size,
-            NULL);
+            NULL,
+            0,
+            &claims,
+            &claims_length);
+    }
+
+    printf("Printing Claim Names\n ");
+    for (size_t j = 0; j < claims_length; j++)
+    {
+        printf("%s \n", claims[j].name);
```

Comment:
Maybe we can show more info in a sample here? Like `printf("%s(%lu)\n", claims[j].name, claims[j].value_size)`.
Just a personal thought.

Reply:
Sure, that should be fine. Added it now